src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/WrappedStaticStateRequestManager.java

```diff
@@ -28,6 +28,6 @@ public WrappedStaticStateRequestManager(@NotNull final IStandardRequestManager w
      */
     public void updateRequestState(@NotNull final IToken token, @NotNull final RequestState state) throws IllegalArgumentException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)